src/pages/index.js

```diff
@@ -56,28 +83,96 @@ const enableValidation = (config) => {
   });
 };
 enableValidation(config);
-
 /**============================================
  *               Functions
  *=============================================**/
+function handleApiCardInfo(cardData) {
+  const cardApiSection = new Section(
+    { items: cardData, renderer: renderCard },
+    cardContainer
+  );
+  cardApiSection.renderItems();
+}
 function renderCard(item, method = "append") {
-  const card = new Card(item, cardSelector, handleImageClick);
+  const card = new Card(
+    item,
+    cardSelector,
+    handleImageClick,
+    handleDeleteClick,
+    handleLikeClick
+  );
   const cardElement = card.renderCard();
   cardSection.addItem(cardElement, method);
 }
 function handleImageClick(data) {
   popupWithImage.open(data);
 }
-function handleProfileSubmit(userData) {
+function handleDeleteClick(card) {
+  openConfirmation.open(card);
+}
+function deleteConfirmed(card) {
+  api
+    .deleteCard(card.cardID)
+    .then(() => {
+      card.cardElement.remove();
+      card.cardElement = null;
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+}
+function handleLikeClick(card, likeAction) {
+  if (likeAction === "addLike") {
+    api.addLike(card._id).catch((err) => {
+      console.error(err);
+    });
+  }
+  if (likeAction === "removeLike") {
+    api.deleteLike(card._id).catch((err) => {
+      console.error(err);
+    });
+  }
+}
+function getInitialApiUserInfo(userData) {
   userInfo.setUserInfo({
-    name: userData.title,
-    description: userData.description,
+    name: userData.name,
+    description: userData.about,
   });
+  userInfo.setUserAvatar(userData.avatar);
+}
+function handleProfileSubmit(userData) {
+  api
+    .setUserInfo(userData.title, userData.description)
+    .then(() => {
+      userInfo.setUserInfo({
+        name: userData.title,
+        description: userData.description,
+      });
+    })
+    .catch((err) => {
+      console.error(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors

Reply:
I like that, simple is better.